JSSQLite.cpp

```diff
@@ -222,7 +230,7 @@ JSAPI_FUNC(sqlite_close) {
 JSAPI_FUNC(sqlite_open) {
     SqliteDB* dbobj = (SqliteDB*)JS_GetInstancePrivate(cx, JS_THIS_OBJECT(cx, vp), &sqlite_db, NULL);
     if (!dbobj->open) {
-        if (SQLITE_OK != sqlite3_open(dbobj->path, &dbobj->db)) {
+        if (SQLITE_OK != sqlite3_open16(dbobj->path, &dbobj->db)) {

```

Comment:
Curious what the difference is between open and open16. But I trust the change makes sense :)

Reply:
Wide char support for path (the lib does the conversion)